packages/wonder-blocks-dropdown/src/components/combobox.tsx

```diff
@@ -684,7 +680,6 @@ export default function Combobox({
                                         testId ? `${testId}-listbox` : undefined
                                     }
                                     aria-label={labels.listbox}
-                                    aria-labelledby={ids.get("input")}
```

Comment:
**question:** @jandrade seems like this was a bug? There's already an `aria-label` attribute, and setting `aria-labelledby` to an ID that isn't attached to any element feels like it'll just confuse screen reader tech and browsers.

Reply:
For the record, there was no bug here, I don't think. We're using label and labelledby, which may be an oversight, but it should be fine as-is.